server/src/test/java/org/elasticsearch/index/query/SimpleQueryStringBuilderTests.java

```diff
@@ -179,11 +190,17 @@ public void testDefaultEmptyComplainFlags() {
     public void testDefaultNullComplainOp() {
         SimpleQueryStringBuilder qb = new SimpleQueryStringBuilder("The quick brown fox.");
         qb.defaultOperator(null);
+        qb.type(null);
         assertEquals(
             "Setting operator to null should result in returning to default value.",
             SimpleQueryStringBuilder.DEFAULT_OPERATOR,
             qb.defaultOperator()
         );
+        assertEquals(
+            "Setting operator to null should result in returning to default value.",
```

Comment:
nit: minor typo here
```suggestion
            "Setting type to null should result in returning to default value.",
```